tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -265,6 +257,13 @@ private void PrepareData(string databaseName, string collectionName, BsonDocumen
             }
         }
 
+        private void RemoveCollectionData(string databaseName, string collectionName)
+        {
+            var database = DriverTestConfiguration.Client.GetDatabase(databaseName);
```

Comment:
Maybe we will have a ticket similar to the current for CRUD tests, but I think there is no harm to add this fix too to avoid fails likes this https://evergreen.mongodb.com/task_log_raw/dot_net_driver_secure_tests__version~latest_os~windows_64_topology~sharded_cluster_auth~auth_ssl~ssl_test_patch_ddbf5052cbf5c1b9bae1118ebf81c05e4029b2ed_5e9e31ea3e8e863a29fd371e_20_04_20_23_36_10/0?type=T#L5577 now.